src/apps/TCO2_Dashboard/app.py

```diff
@@ -0,0 +1,311 @@
+import dash_bootstrap_components as dbc
+import dash
+from dash import html, Input, Output, callback
+from dash import dcc
+from figures import sub_plots_vintage, sub_plots_volume, map, total_vintage, total_volume, \
```

Comment:
The smoke test is failing, I think because all these local module references need to be prefixed with `.` like `from .figures import ...` or `from .tco2 import create_content_toucan`